src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb

```diff
@@ -812,7 +812,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 If ProjectProperties IsNot Nothing Then
                     Return ProjectProperties.OutputType
                 Else
-                    Dim obj As Object = TryGetNonCommonPropertyValue(GetPropertyDescriptor("OutputType"))
+                    Dim obj As Object = TryGetNonCommonPropertyValue(GetPropertyDescriptor("OutputTypeEx"))
```

Comment:
According to the comment that code path is specific to CPS. I'm not sure if this ever worked for CPS projects but what it was doing before was grabbing OutputType while all other places (including the dropdown) are grabbing OutputTypeEx so that's probably a bug in the initial implementation. The summary of these 2 should be - OutputType will return the string for the property (i.e. "Exe") while OutputTypeEx will return the transformation of that into a VSLangProj110.prjOutputTypeEx value.